modules/service/src/main/scala/lucuma/odb/graphql/input/ProgramPropertiesInput.scala

```diff
@@ -7,19 +7,21 @@ package input
 
 import cats.syntax.all._
 import eu.timepit.refined.types.string.NonEmptyString
+import grackle.Result
 import lucuma.odb.data.Existence
+import lucuma.odb.data.Tag
 import lucuma.odb.graphql.binding._
 
 object ProgramPropertiesInput {
 
   case class Create(
     name: Option[NonEmptyString],
-    proposal: Option[ProposalInput.Create],
-    existence: Option[Existence]
```

Comment:
`existence` is not actually used in creation. So, I removed it from here. Users can still specify it in the API, but it is ignored, just as it was before. Since only half the fields in `ProgramPropertiesInput` are used for creation, it makes me wonder if creation and update should have different `Inputs`?

Reply:
I'm confused.  `ProgramPropertiesInput` has

```
  name: NonEmptyString
  proposal: ProposalInput
  existence: Existence
```

We don't use these for program creation?  Is it just that we haven't implemented it yet rather than that there's a fundamental reason why they don't make sense at creation time?

Reply:
`ProgramPropertiesInput` now also has `proposalStatus`. AFAICT, `existence`, although available in `ProgramPropertiesInput.Create` has always just been ignored. I assume because it makes no sense to create a `deleted` program. 
It also makes no sense to create a program with a proposal status of anything except `NOT_SUBMITTED` since it cannot have any observations, yet, having just been created.

Reply:
`Existence` is `PRESENT` vs `DELETED` but maybe should have been `PRESENT` vs `HIDDEN` or something since it is not really deleted.  I could _maybe_ see creating a hidden program and editing it until ready and then making it appear by "undeleting" it.

Reply:
I suppose they could do that via the API (but isn't support in Explore at the moment). If we want to allow that, I could put it back and actually implement it in program creation. Since it wasn't actually being used, I thought it was clearer to remove it from the Create case class.

Reply:
However, it still don't think it makes sense to create one with a non NOT_SUBMITTED proposal class. If it is SUBMITTED, it will be `locked` and will have no observations.